external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -436,11 +438,12 @@ impl fmt::Display for DefInfo {
                     type_to_ide_string(t)
                 )
             }
-            Self::Local(name, t, is_decl) => {
+            Self::Local(name, t, is_decl, is_mut) => {
+                let mut_str = if *is_mut { "mut " } else { "" };
                 if *is_decl {
-                    write!(f, "let {}: {}", name, type_to_ide_string(t))
+                    write!(f, "let {}{}: {}", mut_str, name, type_to_ide_string(t))
```

Comment:
```suggestion
                    write!(f, "let {mut_str}{name}: {}", type_to_ide_string(t))
```

I suppose? Fine keeping the `{}`s for consistencies sake